clear/main.py

```diff
@@ -3,13 +3,13 @@
 import click
 import os
 import subprocess
+import sys
 
-try:
-    # noinspection PyPep8Naming
-    import ConfigParser as configparser
-except ImportError:
-    # noinspection PyUnresolvedReferences
+# TODO: Remove this check once we no longer support Python 2
```

Comment:
No need to remove

Reply:
Why not? Once we no longer support Python 2, there is no need to keep this code around. It's better to clean up obsolete code.

Reply:
I am not longer going to support both Python 2 and Python 3 simultaneously. Since this is a library, there is no point.